pom.xml

```diff
@@ -100,6 +100,13 @@
                 <groupId>com.datastax.cassandra</groupId>
                 <artifactId>cassandra-driver-core</artifactId>
                 <version>${project.parent.version}</version>
+                <classifier>shaded</classifier>
```

Comment:
The shaded classifier doesn't need to be here right?  I see in the individual test poms that it is set / unset explicitly.

Reply:
I need if for the dependency management, because a provided version in dependency management for an artifact with no classifier doesn't work for a dependency with classifier. I can remove the entire dependency from the dependencymanagement in the parent pom and add it (with version) to the test artifact poms. But because you are doing all the dependency management in the main pom I think it's the right point there.